cloud-build/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# This Dockerfile spins up a container where presubmit tests are run.
+# Cloud Build orchestrates this process.
+
+FROM gcr.io/cloud-builders/gcloud
+
+# Copy everything into the container
+COPY . /
```

Comment:
I guess you want to copy the current project to a non / directory, e.g.: `COPY . /dataproc-init-actions/`

Reply:
Done.